android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.novoda.android">
+
+    <application android:allowBackup="true"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
```

Comment:
if they have it set to false they will override this and `false` with be the RTL. If they don't have it set then the default is true like this?